### DISCUSSION TITLE
Is it necessary to fill 'Controlnet model' and it's accompanying fields when enabling Controlnet extension?

### DISCUSSION THREAD

**Discussion by ccbruce0812** [Bing-su/adetailer#174] [Q&A]
I found the dosbox reports NaN if I leave 'Controlnet model' field 'None' when enabling Controlnet extension. Therefore I fill the same model name in 'Controlnet model' in adetailer as the one in Controlnet extension.

However the NaN problem emerges again if 2nd adetailer is enabled with another model such like 'hand_yolov8n.pt'. What is the correct action should I take? Choose the same controlnet model for 2nd adetailer as the one in 1nd adetailer?

Controlnet model in adetailer is very confusing. I literally hope they can cancel it in the feature.

**Comment by Bing-su** [comment]
The NaN issue is related to the model used, Loras, embeddings, and VAE, and has nothing to do with controlnet or the adetailer. Some of them are fine in txt2img but cause problems only in inpainting.

**Reply by Bing-su** [reply]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/6923
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/7633
https://github.com/Bing-su/adetailer/issues/164
